keras_cv/layers/preprocessing/random_mix_up.py

```diff
@@ -0,0 +1,64 @@
+import tensorflow as tf
+import tensorflow.keras as keras
+from tensorflow.keras import layers
+
+
+class RandomMixUp(layers.Layer):
+    def __init__(
+        self,
+        alpha=0.8,
+        probability=1.0,
+        num_classes=None,
+        label_smoothing=0.0,
+        **kwargs
+    ):
+        super(RandomMixUp, self).__init__(*kwargs)
+        if num_classes is None:
+            raise ValueError(
+                "num_classes is required.  Got MixUp.__init__(num_classes=None)"
+            )
+        self.alpha = alpha
+        self.probability = probability
+        self.num_classes = num_classes
+        self.label_smoothing = label_smoothing
+
+    @staticmethod
+    def _sample_from_beta(alpha, beta, shape):
+        sample_alpha = tf.random.gamma(shape, 1.0, beta=alpha)
+        sample_beta = tf.random.gamma(shape, 1.0, beta=beta)
+        return sample_alpha / (sample_alpha + sample_beta)
+
+    def call(self, images, labels):
+        augment_cond = tf.less(
+            tf.random.uniform(shape=[], minval=0.0, maxval=1.0), self.probability
+        )
+        # pylint: disable=g-long-lambda
+        augment_a = lambda: self._update_labels(*self._mixup(images, labels))
```

Comment:
Please always use fully spelled-out variable names to increase readability (e.g. what's "a"?)

Reply:
Went ahead and fixed this - thanks!